CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DSOAView.h

```diff
@@ -0,0 +1,101 @@
+#ifndef CUDADataFormats_TrackingRecHit_interface_TrackingRecHit2DSOAView_h
+#define CUDADataFormats_TrackingRecHit_interface_TrackingRecHit2DSOAView_h
+
+#include <cuda_runtime.h>
+#include <cuda/api_wrappers.h>
+
+#include "CUDADataFormats/SiPixelCluster/interface/gpuClusteringConstants.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
+#include "Geometry/TrackerGeometryBuilder/interface/phase1PixelTopology.h"
```

Comment:
this one should be moved to ```Geometry/CommonTopology``` package

Reply:
@ianna what file do you mean ?
`CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DSOAView.h` or `Geometry/TrackerGeometryBuilder/interface/phase1PixelTopology.h` ?

I would be hesitant to move the first one, because of the dependency on CUDA.

We can certainly move the second one - but it's not a new file introduced by this PR, it is already present in release.

Reply:
@ianna could you remind me what you meant ?